rules/preprocessing.smk

```diff
@@ -0,0 +1,124 @@
+import os
+import glob
+# assimptions:
```

Comment:
typo

also, are these notes specific to the file, or your personal debugging notes? It looks like these are specific to the file, but the ones at the end of this file are not. Please delete the extra lines to clean things up! 🧹

Reply:
Typo fixed and removed the notes at the end of these files. The notes above the  documentation of the DAG. 

I removed the unnecessary notes and improved the documentation explaining what the DAG is doing.  

commit: d74575671a20747cfe0ac21744b162584cc915de

Feel free to unresolved this thread if more work needs to be done on the documentation. I do not have a general structure on how the documentation should be written in order to maintain readability.